tests/test_benchmark.py

```diff
@@ -388,39 +345,21 @@ def test_hif2a():
 def test_nonbonded_interaction_group_potential(hi2fa_test_frames):
     pots, sys_params, frames, boxes, ligand_idxs = hi2fa_test_frames
     lambdas = np.array([0.0, 1.0])
-    nonbonded_interp = pots[3]
-    assert isinstance(nonbonded_interp, NonbondedInterpolated)
+    U_names = [type(U_fn).__name__ for U_fn in pots]
+    nb_idx = U_names.index("Nonbonded")
```

Comment:
nice! I was just doing `pots[-1]` - I like your approach better.

Reply:
nit: I think this also works

```python
nonbonded = next(p for p in pots if isinstance(p, Nonbonded))
```

Reply:
Sure, but I need the index for sys_params too.